### ISSUE TITLE
Quality is not same as original device 

### ISSUE THREAD

**Issue by Bhoomishah1990** [gmerwan/cameraview#2]
I have tested some device. but Quality is not same as original device.
i have captured photo from Device. and also capture using this lib.
Both quality is diffrent. (Not same as Original)

**Comment by gmerwan** [comment]
Yes, the quality between the phone stock camera app and using this library won't be the same.
But you can tweak this library and modify parameters in order to get the best quality.

If you wanna more built-in choices try to use the CameraKit repo, it's so good.
